### ISSUE TITLE
Error since 0.4.2, " Failed to execute 'postMessage' on 'BroadcastChannel': #<Object> could not be cloned."

### ISSUE THREAD

**Issue by sk-tjdownes** [wobsoriano/pinia-shared-state#27]
First of all, let me say thank you for creating and maintaining this awesome project. It saved our behinds on a pinia store issue where we use `pinia-plugin-persisted-state` to cache the stores in localstorage, and users were logging into two separate accounts in different tabs, corrupting the stores states!

I'm using Vue 3.3.4 with pinia 2.1.4. This exception started appearing recently. I believe it was today after updating to 0.4.2 from 0.3.0. It's repeatable on the `vue-date-picker` component inside a vuetify dialog, for us. Whenever the date value changes the exception occurs. To add to this, we use rrule and computed properties to calculate quite a few things, so it could be anywhere in that chain.

I can't see any immediate impact to our app, but I suspect that something may be wrong with the shared state. For now it's not urgent for us, since we have no side effects. Let me know if you need anything else to help debug.

```
native.js:24 Uncaught (in promise) DOMException: Failed to execute 'postMessage' on 'BroadcastChannel': #<Object> could not be cloned.
    at Object.postMessage (http://localhost:5173/node_modules/.vite/deps/pinia-shared-state.js?v=24bbf7a2:67:21)
    at http://localhost:5173/node_modules/.vite/deps/pinia-shared-state.js?v=24bbf7a2:760:47
postMessage @ native.js:24
(anonymous) @ broadcast-channel.js:187
Show 2 more frames

```

**Comment by wobsoriano** [comment]
Hi, thanks for reporting! Glad you liked it.

Can you try 0.4.4?

**Comment by HaminJuri** [comment]
I tried 0.4.4 but still getting same error as mentioned above,(url)
![Untitled-min](https://github.com/wobsoriano/pinia-shared-state/assets/137410597/813f14ec-55e1-4a14-b4a3-08d818e91963)

**Comment by sk-tjdownes** [comment]
> Can you try 0.4.4?

@wobsoriano I am also still getting the same error
![image](https://github.com/wobsoriano/pinia-shared-state/assets/132268438/810380a3-0131-4c37-82a8-b093b89c94ad)

**Comment by wobsoriano** [comment]
@sk-tjdownes but when you go back to 0.3.0, it works?

**Comment by sk-tjdownes** [comment]
@wobsoriano I just rolled back to `0.3.0` and tested, no errors

**Comment by wobsoriano** [comment]
Thanks @sk-tjdownes! Should be working now. Rolled back to 0.3.0 proxy unwrapping.

**Comment by HaminJuri** [comment]
Thank You, I've been reading your conversations like an Spy :)
BTW, this update you released works well🫡

**Comment by sk-tjdownes** [comment]
@wobsoriano this appears resolved in 0.4.5. Thanks for your speedy attention!
